api/Animation.json

```diff
@@ -374,13 +374,13 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Animation/effect",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "75"
             },
             "chrome_android": {
-              "version_added": false
+              "version_added": "75"
             },
             "edge": {
-              "version_added": false
+              "version_added": "≤79"
```

Comment:
I stuck with the ranged versions already used in Animation and KeyframeEffect, but have verified no support for these or AnimationEffect in Edge 18. Happy to fix the data, but would prefer to not touch too much in a single PR.